error: invalid use of incomplete type ‘class QDebug’ in `example/main.cpp`

System is Ubuntu 20.04-based (elementary OS Jólnir [6.1]) with all required development packages.

<details>
    <summary>The <code>make install</code> fails with the following error messages: (click to expand)</summary>
<pre><code>$ make
[ 11%] Automatic MOC for target ksnip-plugin-ocr
[ 11%] Built target ksnip-plugin-ocr_autogen
[ 55%] Built target ksnip-plugin-ocr
[ 66%] Automatic MOC for target plugin-ocr-example
[ 66%] Built target plugin-ocr-example_autogen
[ 77%] Building CXX object example/CMakeFiles/plugin-ocr-example.dir/main.cpp.o
/home/me/src/graphics/ksnip-plugin-ocr/example/main.cpp: In function ‘int main(int, char**)’:
/home/me/src/graphics/ksnip-plugin-ocr/example/main.cpp:77:12: error: invalid use of incomplete type ‘class QDebug’
   77 |     qDebug() << "Error loading plugin" << fileName << pluginLoader.errorString();
      |            ^
In file included from /usr/include/x86_64-linux-gnu/qt5/QtCore/qglobal.h:1206,
                 from /usr/include/x86_64-linux-gnu/qt5/QtGui/qtguiglobal.h:43,
                 from /usr/include/x86_64-linux-gnu/qt5/QtWidgets/qtwidgetsglobal.h:43,
                 from /usr/include/x86_64-linux-gnu/qt5/QtWidgets/qapplication.h:43,
                 from /usr/include/x86_64-linux-gnu/qt5/QtWidgets/QApplication:1,
                 from /home/me/src/graphics/ksnip-plugin-ocr/example/main.cpp:20:
/usr/include/x86_64-linux-gnu/qt5/QtCore/qlogging.h:57:7: note: forward declaration of ‘class QDebug’
   57 | class QDebug;
      |       ^~~~~~
/home/me/src/graphics/ksnip-plugin-ocr/example/main.cpp:79:12: error: invalid use of incomplete type ‘class QDebug’
   79 |     qDebug() << "Checked" << fileName << "and it's not a plugin.";
      |            ^
In file included from /usr/include/x86_64-linux-gnu/qt5/QtCore/qglobal.h:1206,
                 from /usr/include/x86_64-linux-gnu/qt5/QtGui/qtguiglobal.h:43,
                 from /usr/include/x86_64-linux-gnu/qt5/QtWidgets/qtwidgetsglobal.h:43,
                 from /usr/include/x86_64-linux-gnu/qt5/QtWidgets/qapplication.h:43,
                 from /usr/include/x86_64-linux-gnu/qt5/QtWidgets/QApplication:1,
                 from /home/me/src/graphics/ksnip-plugin-ocr/example/main.cpp:20:
/usr/include/x86_64-linux-gnu/qt5/QtCore/qlogging.h:57:7: note: forward declaration of ‘class QDebug’
   57 | class QDebug;
      |       ^~~~~~
make[2]: *** [example/CMakeFiles/plugin-ocr-example.dir/build.make:76: example/CMakeFiles/plugin-ocr-example.dir/main.cpp.o] Error 1
make[1]: *** [CMakeFiles/Makefile2:199: example/CMakeFiles/plugin-ocr-example.dir/all] Error 2
make: *** [Makefile:130: all] Error 2</code>
</details>

Solution was to add

```c++
#include <QDebug>
```

to `example/main.cpp`.

A PR is forthcoming.

--- a/example/main.cpp
+++ b/example/main.cpp
@@ -24,6 +24,7 @@
 #include <QPainter>
 #include <QLabel>
 #include <QVBoxLayout>
+#include <QDebug>
 
 #include "IPluginOcrOther.h"
 

